.sbtopts

```diff
@@ -0,0 +1,5 @@
+-J-Xss2M
```

Comment:
Super pedantic, but can we move this flag below the comments to be consistent with the code style in rest of the build scripts? (Wouldn't bother to ask but I'm already marking a different change :-D )

Reply:
yes of course! I forgot to change that back, it looks odd as is